### PR TITLE
Disconnect peers with version < 70212 after DIP3 activation via BIP9

### PR DESCRIPTION
**Pull request by codablock**

### CODE REVIEW

File: src/net_processing.cpp [hunk 1]
```diff
@@ -1564,6 +1577,13 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
     // At this point, the outgoing message serialization version can't change.
     const CNetMsgMaker msgMaker(pfrom->GetSendVersion());
 
+    // BEGIN TEMPORARY CODE
```
Comment (UdjinM6) [inline src/net_processing.cpp]: This part is not needed imo.

Comment (codablock) [reply src/net_processing.cpp]: This is for nodes which were connected before DIP3 BIP9 activation as they won't run into the version check otherwise.

Comment (codablock) [reply src/net_processing.cpp]: done

**Comment by PastaPastaPasta** [comment]
Doesn't this have the potential to create an unintentional fork if there is a pool/exchange/user which isn't upgraded yet (but BIP9 activation has occurred) as they won't get relayed the correct blocks since they can't have upgraded peers?

**Comment by codablock** [comment]
Such peers would fork away nevertheless at the next block after Dip3 activation. The coinbase special TX alone would cause this. So the conclusion is that these nodes become completely useless at that point and thus can be disconnected

**Pull request merged**